deps/Makefile

```diff
@@ -178,7 +178,17 @@ install: $(addprefix install-, $(DEP_LIBS))
 cleanall: $(addprefix clean-, $(DEP_LIBS))
 distcleanall: $(addprefix distclean-, $(DEP_LIBS))
 	rm -rf $(build_prefix)
-getall: get-llvm get-libuv get-pcre get-openlibm get-dsfmt get-openblas get-lapack get-suitesparse get-unwind get-osxunwind get-gmp get-mpfr get-patchelf get-utf8proc get-objconv get-mbedtls get-libssh2 get-curl get-libgit2 get-libwhich
+getall: get-llvm get-libuv get-pcre get-openlibm get-dsfmt get-openblas get-lapack get-suitesparse get-unwind get-gmp get-mpfr get-patchelf get-utf8proc get-objconv get-mbedtls get-libssh2 get-curl get-libgit2 get-libwhich
+
+# If we're building for MacOS, no matter what, `getall` should include `osxunwind`
+ifeq ($(OS),Darwin)
+getall: get-osxunwind
```

Comment:
It's been a while since I've Makefile'd. Does this redefine the `getall` target or does it add the new prerequisite to the existing target?

Reply:
The latter

Reply:
Neato, TIL.